src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -137,6 +138,59 @@ private string ResolveMatchedPath(string matchedPath)
             return RequestPathBase.Add(relativePath).ToString();
         }
 
+        private class PathComparer : IComparer<string>
+        {
+            public int Compare(string x, string y)
+            {
+                // < 0 = x < y
+                // > 0 = x > y
+
+                if (x.Equals(y))
+                {
+                    return 0;
+                }
+
+                var xExtIndex = x.LastIndexOf('.');
+                var yExtIndex = y.LastIndexOf('.');
```

Comment:
I was wondering about that myself. I'll see if the `Matcher` API matches them and if so adjust this accordingly

-----Original Message-----
From: "Eilon Lipton" <notifications@github.com>
Sent: ‎2/‎24/‎2015 8:52 AM
To: "aspnet/Mvc" <Mvc@noreply.github.com>
Cc: "Damian Edwards" <damian@damianedwards.com>
Subject: Re: [Mvc] Script & Link tag helpers should do explicit ordering ofglobbed path results (#2067)

In src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs:
> @@ -137,6 +138,59 @@ private string ResolveMatchedPath(string matchedPath)
>              return RequestPathBase.Add(relativePath).ToString();
>          }
>  
> +        private class PathComparer : IComparer<string>
> +        {
> +            public int Compare(string x, string y)
> +            {
> +                // < 0 = x < y
> +                // > 0 = x > y
> +
> +                if (x.Equals(y))
> +                {
> +                    return 0;
> +                }
> +
> +                var xExtIndex = x.LastIndexOf('.');
> +                var yExtIndex = y.LastIndexOf('.');
Can there be files without file extensions?
—
Reply to this email directly or view it on GitHub.